bot/interact.go

```diff
@@ -148,15 +148,15 @@ func generateSubmitJobFormDialog() slack.Dialog {
 	salaryCurrencyInput.Optional = false
 	salaryCurrencyInput.Hint = "Choose the salary currency from the dropdown"
 
-	salaryMinInput := slack.NewTextInput("min_salary", "Salary min", "")
+	salaryMinInput := slack.NewTextInput("min_salary", "Salary min (per year)", "")
```

Comment:
Maybe this should be in the hint?

Reply:
I think it has more visibility as it is now. The hint is already big enough.